ICAFIX/scripts/RecleanClassify.sh

```diff
@@ -0,0 +1,284 @@
+        mkdir -p ${WorkingDirectory}
+        cp ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_dropouts.nii.gz ${WorkingDirectory}/${fMRIName}_dropouts.nii.gz
+        gunzip -f ${WorkingDirectory}/${fMRIName}_dropouts.nii.gz
+        "$HCPPIPEDIR"/global/scripts/MapVolumeToCIFTI.sh ${StudyFolder} ${Subject} ${fMRIName} ${CorticalLUT} ${SubCorticalLUT} ${Caret7_Command} ${LowResMesh} ${RegName} ${SmoothingFWHM} ${FinalfMRIResolution} ${BrainOrdinatesResolution} ${WorkingDirectory}/${fMRIName}_dropouts.nii ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_dropouts.dscalar.nii ${fMRIName}_dropouts ${Flag} ${DeleteIntermediates} nii ${WorkingDirectory}
+        rm -r ${WorkingDirectory}
+        fi
+    done
+fi
+
+# compute additional features from matlab script
+#FIXME: parcellation files are saved under tICA folder--reclassify-as-signal-file
+CorticalParcellationFile="$HCPPIPEDIR/global/templates/tICA/Q1-Q6_RelatedValidation210.CorticalAreas_dil_Final_Final_Areas_Group_Colors.32k_fs_LR.dlabel.nii"
+SubRegionParcellation="$HCPPIPEDIR/global/templates/tICA/Q1-Q6_RelatedParcellation210.CorticalAreasAndSubRegions_dil.32k_fs_LR.dlabel.nii"
+WMLabelFile="$HCPPIPEDIR/global/config/FreeSurferWMRegLut.txt"
+CSFLabelFile="$HCPPIPEDIR/global/config/FreeSurferCSFRegLut.txt"
+VisualAreasFile="$HCPPIPEDIR/global/config/Visual.ROI.txt"
+LanguageAreasFile="$HCPPIPEDIR/global/config/Language.ROI.txt"
+SubRegionsFile="$HCPPIPEDIR/global/config/SubRegions.ROI.txt"
+NonGreyParcelsFile="$HCPPIPEDIR/global/config/NonGreyParcels.ROI.txt"
+GrayOrdinateTemplateFile="$HCPPIPEDIR/global/templates/91282_Greyordinates/91282_Greyordinates.dscalar.nii"
+
+tempfiles_create MMPROIs_temp_XXXXXX.dlabel.nii roitemp
+wb_command -cifti-create-dense-from-template "$HCPPIPEDIR"/global/templates/91282_Greyordinates/91282_Greyordinates.dscalar.nii "$roitemp" -cifti "$CorticalParcellationFile"
+
+tempfiles_create MMPSubRegionROIs_temp_XXXXXX.dlabel.nii roisubtemp
+wb_command -cifti-create-dense-from-template "$HCPPIPEDIR"/global/templates/91282_Greyordinates/91282_Greyordinates.dscalar.nii "$roisubtemp" -cifti "$SubRegionParcellation"
+
+tempfiles_create VisualROI_temp_XXXXXX.dlabel.nii VisualROItemp
+tempfiles_add "$VisualROItemp"_labels.dlabel.nii
+wb_command -cifti-label-import "$roitemp" "$HCPPIPEDIR"/global/config/Visual.ROI.txt "$VisualROItemp"_labels.dlabel.nii -discard-others
+wb_command -cifti-math 'x > 0' "$VisualROItemp" -var x "$VisualROItemp"_labels.dlabel.nii
+
+tempfiles_create LanguageROI_temp_XXXXXX.dlabel.nii LanguageROItemp
+tempfiles_add "$LanguageROItemp"_labels.dlabel.nii
+wb_command -cifti-label-import "$roitemp" "$HCPPIPEDIR"/global/config/Language.ROI.txt "$LanguageROItemp"_labels.dlabel.nii -discard-others
+wb_command -cifti-math 'x > 0' "$LanguageROItemp" -var x "$LanguageROItemp"_labels.dlabel.nii
+
+# shortcut in case the folder gets renamed
+this_script_dir=$(dirname "$0")
+HelpFuncPath="$this_script_dir/scripts"
```

Comment:
This script is in the `scripts` folder, this path will be `ICAFIX/scripts/scripts`, which is almost certainly wrong.  But, it looks like `HelpFuncPath` isn't used, so why does it exist?